packages/react-instantsearch/src/connectors/connectRange.js

```diff
@@ -1,4 +1,5 @@
 import PropTypes from 'prop-types';
+import { isFinite as _isFinite } from 'lodash';
```

Comment:
By curiosity, why using lodash here instead of the native one?

Reply:
The global `isFinite` (see [reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite)) function is bit broken. For example:

```js
isFinite(null) // true
```

We should use `Number.isFinite` (see [reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite)), but the support is not so great! The `isFinite` function from `lodash` is based on `Number.isFinite` (see [docs](https://lodash.com/docs/4.17.4#isFinite)).